.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -31,7 +31,12 @@ Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerc
 
 ### Changelog entry
 
-<!-- You can optionally choose to enter a changelog entry by checking the box and supplying data. -->
+<!-- You can optionally choose to enter a changelog entry by checking the box below and supplying data. -->
+<!-- It will trigger the 'Add changelog to PR' CI job to create and push the entry into the branch. -->
+
+<!-- Due to org permissions, the job might fail for PRs crated from a fork under GitHub organizations. Possible solutions: -->
+<!-- * Create entry manually with `pnpm --filter='package-name-here' changelog add` and push it into the branch -->
```

Comment:
You can ignore this, I'm just trying to put myself in the shoes of contributors. How do they determine what `package-name-here` should be?

I guess we don't want to expand this to cover every possible use case (so, again, don't hesitate to dismiss this suggestion) but I wonder if we should amplify this with some examples ... `@woocommerce/plugin-woocommerce` is *probably* the most commonly used package name (maybe)?

Reply:
No worries, more clarity will not hurt. Here we go: 1c3826df93d923256ec657e5baccb6094b92aff2, my best short for clarity and compact wording.